### ISSUE TITLE
[CAIA A11y Research]: Modernized Check-in, Unified Check-in Experience

### ISSUE THREAD

**Issue by cgednalske** [department-of-veterans-affairs/va.gov-team#58566]
### Who's on your team?

- Team name: Modernized Check-in
- OCTO-DE product owner: Stephen Barrs
- Product manager: Lori Pusey
- Researcher/moderator: Ben Brasso (secondary moderator: Christina Gednalske )
- Designers: Zach Park, Ya-Ching Tsao, Christina Gednalske
- Content Designer: Kelly Smith
- FE engineers: Brian Seek, John Woolschlager
- Product/team Slack channel: check-in-experience

...

#### Do you have an accessibility champion on your team?

No, we don't have a champion (yet)

### Calendar checking

- [X] Yes, we have reviewed the[ CAIA capacity calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/newyork) and can confirm specialists are available to support in our research timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### When is the timeframe for your testing?

June 12th - 20th (not 19th)

## Prototype
[Pre-check-in start](https://codepen.io/team/va-cie/live/NWORQEe)
[Check-in start](https://codepen.io/team/va-cie/live/qBJqWaV)

### Device
- All participants should be joining on their mobile phones.
- iPhone only was requested.

### Monday, June 12
**Participant 1 — AT (screen reader)** `No Session`
11am - 12:30pm EST
- AT Support: Angela Fowler
- AT Info: iPhone accessibility features, Intermediate experience level
	
**Participant 2 — AT (screen reader)** `Completed Session`
1pm - 2:30pm EST
- AT Support: Sara Smith
- AT Info: VoiceOver, Intermediate experience level

### Wednesday, June 14
**Participant 8 — AT (screen reader)** `No Session`
11am - 12:30pm EST
- AT Support: Sara Smith and/or SK
- AT Info: VoiceOver, Advanced experience level

**Participant 10 — AT (screen reader)** `No Session`
3pm - 4:30pm EST
- AT Support: Angela Fowler and/or SK
- AT Info: VoiceOver, Intermediate experience level

### Thursday, June 15
**Participant 11 — AT (screen magnification)**
9am - 10am EST
- [AT Support:](https://github.com/department-of-veterans-affairs/va.gov-team/issues/58566#issuecomment-1584842643) @SarahKay8 
- AT Info: Screen magnification tool, Expert experience level

**Participant 12 — AT (screen reader)**
11am - 12:30pm EST
- AT Support: Angela Fowler and/or SK
- AT Info: VoiceOver, Advanced experience level

### Tuesday, June 20
**Participant 17 — AT (screen magnification)**
9am - 10am EST
- AT Support: Sara Smith
- AT Info: Screen magnification tool, Beginner experience level

## Supporting artifacts (if any)

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/product) (general - not specific to unified check-in)
- [User flows](https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec)
- CodePen prototype: [Pre-check-in start](https://codepen.io/team/va-cie/live/NWORQEe), [Check-in start](https://codepen.io/team/va-cie/live/qBJqWaV) (view in mobile size or mobile inspect mode)
- [Mockups](https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/p/0EC89917-F949-4461-A7B3-32A5201FD2A2/canvas)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/2023-04%20Unified%20Check-In%20Experience%20Usability%20Study/research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/2023-04%20Unified%20Check-In%20Experience%20Usability%20Study/conversation-guide.md)
- [Epics for unified check-in](https://github.com/department-of-veterans-affairs/va.gov-team/issues/49924)
...

## Next steps

**Complete these after submitting this ticket with your CAIA specialist**

- [x] Post a link to this ticket in the [accessibility-help Slack channel and tag Terry Nichols](https://dsva.slack.com/archives/C8E985R32).
- [x] Your team's researchers and observers [read through platform guidance on research with assistive technology users](https://depo-platform-documentation.scrollhelp.site/research-design/research-with-assistive-technology-users)
- [x] CAIA Specialists are assigned to your research
- [x] CAIA Specialists review your conversation guide and research plan
    - [x] **RCVD** [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/2023-04%20Unified%20Check-In%20Experience%20Usability%20Study/research-plan.md)
    - [x] **RCVD** [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/2023-04%20Unified%20Check-In%20Experience%20Usability%20Study/conversation-guide.md)
- [x] Provide specialists your participant details like skill level, operating system, assistive tech used, browser. _(Refer to the "Confirmed Sessions" section above.)_
- [ ] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [x] CAIA Specialists create a[ research notes template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/research/accessibility-findings-template.md)
- [x] CAIA Post research, specialists will add accessibility findings to your research folder

**Comment by sara-amanda** [comment]
## CAIA Update

- Adding links to the next steps section
- Adding the details needed when participant information is known
- Will plan to sign up for AT sessions I am available for once posted

cc: @SarahKay8 @coforma-terry @cgednalske @benbrasso-agile6

**Comment by cgednalske** [comment]
Hi @sara-amanda! Could you give a little more detail about your comment above? Are any of those items something we need to do on our end?

Also, would you be able to give us an idea of your general availability during the testing timeframe - like some blocks of time you know you'll be free? If we can know more or less when you're available, we'll use that as the starting point for scheduling screen readers sessions, so it's more likely our schedules will line up.

We also plan to reach out to Angela Fowler to see about her availability for additional tech assistance when you're not free. We're hoping that between the two of you it will be easy to have AT tech support covered throughout the study. 🤞 

Thanks so much!

**Comment by sara-amanda** [comment]
Hi @cgednalske! 

I can add some more details below, relating to those next steps, for you to delineate CAIA vs. your team. 😃
 
## Remaining Next Steps for Your Team
⚪  **CAIA** 🟢  **Your** **Team**

- [ ] CAIA Specialists are assigned to your research. **(🟢 Your Team)** 
    - You will want to invite CAIA Specialists (@sara-amanda and @SarahKay8) to your research and reference the [CAIA capacity calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/newyork) for scheduling purposes.
    - Specifically, I am available (as of 5/17/2023) during your testing dates June 12 - 20:
        - Monday and Tuesday June 12 and 13 from 9 a.m. - 4 p.m. EST
        - Wednesday, June 14, between 9 a.m. and 1 p.m. EST.
        - Tuesday, June 20 from 9 a.m. - 2 p.m. EST and 3 to 5 p.m. EST. 
   - **Yes**! Please do include @AngelaFowler82, if she is available. 
       - We can have join sessions as well with her, as we have with other teams, should schedules permit.  
- [ ]  CAIA Specialists review your conversation guide and research plan **(🟢 Your Team & ⚪CAIA)** 
    - It looks like you provided the links to these in the ticket above, so we will review them prior to your testing sessions (_thank you!_):
        - [x] **Received:** [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/2023-04%20Unified%20Check-In%20Experience%20Usability%20Study/research-plan.md)
        - [x] **Received:** [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/2023-04%20Unified%20Check-In%20Experience%20Usability%20Study/conversation-guide.md)
- [ ]  Provide specialists your participant details like skill level, operating system, assistive tech used, browser **(🟢Your Team c/o Perigean data provided to you)**  
    - We are collecting this information to assist in the analysis of the data. 
    -  **Participant #**
        - Day of the week: 
        - Date of the session:
        - Time of the session and timezone:
        - Assistive Tech:
        - AT Skill Level:
        - Operating System:
        - Browser:
        - Device Type:
- [ ]  CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future. **(CAIA ⚪)**
- [ ]  CAIA Specialists create a[ research notes template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/research/accessibility-findings-template.md) **(CAIA ⚪)**
- [ ]  CAIA Post research, specialists will add accessibility findings to your research folder **(CAIA ⚪)**

**_Let me know if you have any questions. Happy to help!_** 🙌

**Comment by cgednalske** [comment]
@sara-amanda wow, thank you! This is SUPER helpful 😃 Thanks so much for providing your availability. That will really help us with scheduling. Also heard from Angela today and she is checking on her availability as well 🎉  

We'll go through everything and let you know if we have questions!

**Comment by cgednalske** [comment]
Hi @sara-amanda 👋 Our materials have just been approved and submitted to Perigean, so you should start to receive session invitations soon. We will also provide our participant list once we receive it. 

**One note:** Angela Fowler will be available to support some sessions too. 🎉  So if you receive an invite outside of the times you listed above, those sessions represent some additional times Angela provided.

**Comment by cgednalske** [comment]
@sara-amanda For reference, here's a schedule of all the AT sessions (confirmed via Slack) and related AT info we've received. All participants should be joining on their mobile phones (iPhone only was requested). 

In addition, here is our prototype, if you'd like to go through to get more familiar.
[Pre-check-in start](https://codepen.io/team/va-cie/live/NWORQEe)
[Check-in start](https://codepen.io/team/va-cie/live/qBJqWaV)

cc @AngelaFowler82 

### Monday, June 12
**Participant 1 — AT (screen reader)**
11am - 12:30pm EST
- AT Support: Sara Smith
- AT Info: iPhone accessibility features, Intermediate experience level
	
**Participant 2 — AT (screen reader)**
1pm - 2:30pm EST
- AT Support: Sara Smith
- AT Info: VoiceOver, Intermediate experience level

### Wednesday, June 14
**Participant 8 — AT (screen reader)**
11am - 12:30pm EST
- AT Support: Sara Smith
- AT Info: VoiceOver, Advanced experience level

**Participant 10 — AT (screen reader)**
3pm - 4:30pm EST
- AT Support: Angela Fowler
- AT Info: VoiceOver, Intermediate experience level

### Thursday, June 15
**Participant 11 — AT (screen magnification)**
9am - 10am EST
- AT Support: _No one available_
- AT Info: Screen magnification tool, Expert experience level

**Participant 12 — AT (screen reader)**
11am - 12:30pm EST
- AT Support: Angela Fowler
- AT Info: VoiceOver, Advanced experience level

### Tuesday, June 20
**Participant 17 — AT (screen magnification)**
9am - 10am EST
- AT Support: Sara Smith
- AT Info: Screen magnification tool, Beginner experience level

**Comment by SarahKay8** [comment]
Hi @cgednalske I can assist with Participant sessions 10-12.

**Comment by sara-amanda** [comment]
@cgednalske updated your list above adding SK's availability.

**Comment by sara-amanda** [comment]
## A11y Research Findings
c/o @sara-amanda @SarahKay8 @AngelaFowler82 
- **GitHub Path:** products/health-care/checkin/research/2023-04 Unified Check-In Experience Usability Study/accessibility-findings-mhv-pci.md
- [GitHub Link to PCI Research Findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/2023-04%20Unified%20Check-In%20Experience%20Usability%20Study/accessibility-findings-mhv-pci.md)

cc: @benbrasso-agile6 @cgednalske
